packages/core/rollup.config.js

```diff
@@ -119,6 +121,16 @@ export default {
             ? generateCssModulesMockName(name)
             : generateCssModulesScopedName(name, filename, css)
       }
+    }),
+    copy({
```

Comment:
if you decide it's cool to remove both suggested in the bottom, don't forget to add a task to remove also the copy plugin

Reply:
Yeah I'll take all the colors issues to a different PR 👍🏼